PrestaShop/autoupgrade
# Add dynamic php requirements check for upgrade

| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Adds a new requirement, regarding  php compatibility of the target version in the upgrade process
| Type?             | improvement
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | -
| Sponsor company   | -
| How to test?      | see below

1. Try to update to a version with the correct PHP version. No alerts or errors should be visible

2. Try to update to a version requiring another version of PHP. An alert must be present specifying the compatibility range of the requested PHP versions.

4. Try to update to version 9, a warning should be present specifying that the PHP requirements could not be found
